### DISCUSSION TITLE
Audio Track Automation?

### DISCUSSION THREAD

**Discussion by opeologist** [SynthstromAudible/DelugeFirmware#893] [Ideas]
i love the new automation work done in the community firmware, but was surprised to see its absence for audio tracks. is there any discussion already related to adding this functionality to audio tracks? thank you!

my first thought related is for effects on an audio track, but also things like pitch automation would be amazing

**Comment by seangoodvibes** [comment]
It was out of scope for the first release but I plan on working on it soon to be included in the next release

**Reply by opeologist** [reply]
awesome!! thanks so much for letting me know, i can't wait for the new release! 🙌

**Reply by miaouxmiaoux** [reply]
Thanks, was just logging in to request this
